packages/astro/test/fixtures/astro-get-static-paths/src/pages/pizza/[...pizza].astro

```diff
@@ -0,0 +1,22 @@
+---
+export function getStaticPaths() {
+	return [{
+		params: { pizza: 'papa-johns' },
+	}, {
+		params: { pizza: 'dominos' },
+  }, {
+		params: { pizza: 'grimaldis/new-york' },
```

Comment:
```suggestion
		params: { pizza: 'loumalnatis/chicago' },
```

Just kidding! 😅

Reply:
WOW OKAY I SEE HOW IT IS. Might have to add `loumalnatis/chicago` to a 404 test bc _that's_ not making the list 😉